src/serialize_sequence.cc

```diff
@@ -110,10 +110,10 @@ std::ostream& operator<<(std::ostream& stream, const Step& step)
         << std::put_time(std::localtime(&execution), "%Y-%m-%d %H:%M:%S") << '\n';
 
     stream << "-- timeout: ";
-    if ( step.get_timeout() == Step::infinite_timeout )
+    if (step.get_timeout().is_infinite())
         stream << "infinite\n";
     else
-        stream << step.get_timeout().count() << '\n';
+        stream << static_cast<std::chrono::milliseconds>(step.get_timeout()).count() << '\n';
```

Comment:
Would it be confusing or good to add `Timeout::count()` that returns this?

We have the floating point conversion functions, and maybe that as integer output?

Reply:
I don't like `count()` without an explicit type because you never know what it refers to:
```
using Duration = std::chrono::milliseconds;
Duration a = how_long_does_the_earth_take_to_orbit_the_sun();
std::cout << a.count() << " milliseconds\";
```
... will produce wrong output once someone changes the typedef to something else, without any compiler warning.